tensorrt_llm/_torch/models/modeling_qwen2vl.py

```
@@ -77,6 +77,41 @@
+    import copy
+
+    if getattr(processor, "_merge_kwargs_cached_installed", False):
+        return
+
+    cache: dict = {}
+    orig = processor._merge_kwargs
+
+    def _cached_merge_kwargs(*args, **kwargs):
+        try:
+            key = repr(sorted(kwargs.items()))
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)